ansible_collections/arista/avd/roles/eos_designs/docs/tables/node-type-structure.md

```diff
@@ -37,9 +58,36 @@
               # The Node Name is used as "hostname".
             - name: <str; required; unique>
 
+              # Specify range of port-ids to reserve for port-channels.
```

Comment:
This should trigger a concern. This is the only key added to this table. The table is only supposed to be about the structure itself, so not have any specific keys.